### ISSUE TITLE
Crash when using clickable area without defining key

### ISSUE THREAD

**Issue by birjolaxew** [jaagr/lemonbuddy#104]
Lemonbar's clickable areas support not defining the key to listen for, in which case it defaults to left click:

    %{A:reboot:}Test{%A}

In lemonbuddy, this instead results in a crash with the error:

    Error in `lemonbuddy': double free or corruption (out)

If you define the key to listen for, it works fine:

    %{A1:reboot:}Test{%A}

---

Relevant configuration:

    [module/test]
    type = custom/script
    exec = echo "%{A:reboot:}Test%{A}"

Full error (minus memory map):

```
*** Error in `lemonbuddy': double free or corruption (out): 0x00007fa710000f40 ***
======= Backtrace: =========
/usr/lib/libc.so.6(+0x70c4b)[0x7fa72718dc4b]
/usr/lib/libc.so.6(+0x76fe6)[0x7fa727193fe6]
/usr/lib/libc.so.6(+0x777de)[0x7fa7271947de]
lemonbuddy(_ZN10lemonbuddy6v2_0_03bar5parseENSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEb+0x249)[0x496259]
lemonbuddy(_ZN10lemonbuddy6v2_0_010controller16on_module_updateENSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE+0xa2d)[0x49730d]
lemonbuddy(_ZNK8delegate7Signal1INSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEE4emitES6_+0xb1)[0x47cde1]
lemonbuddy(_ZN10lemonbuddy6v2_0_07modules12timer_moduleINS1_13script_moduleEE6runnerEv+0x15b6)[0x4b7546]
/usr/lib/libstdc++.so.6(+0xbb31f)[0x7fa7279aa31f]
/usr/lib/libpthread.so.0(+0x7454)[0x7fa7274c2454]
/usr/lib/libc.so.6(clone+0x5f)[0x7fa7272057df]
======= Memory map: ========
...
```

**Comment by jaagr** [comment]
Thanks for reporting it.. Seems like you found two bugs in one there, so that's nice! I noticed in the code that I forgot to implement the default fallback when rewriting 2.0... So the error is caused by the app trying to close a non-existing action. I'll push a fix in a few minutes.

**Comment by jaagr** [comment]
Fixed in de0c3eb46adc78a901f6fcfbba727e41b7f51098, please verify.
